### ISSUE TITLE
Homebridge NOT RESPONDING (after some time) - Fixed - Check your (Netgear) Switch!

### ISSUE THREAD

**Issue by christiantschoerner** [homebridge/homebridge#2763]
Hey Homebridge Users,

**THIS ERROR CAN IMMEDIATELY BE MARKED AS FIXED**

I switched from a Raspberry Pi Zero (on Wifi) to a Raspberry Pi 4 on Ethernet and Restored my old configuration. All worked fine for serveral minutes and then devices started that "No Response" thing that everybody just hates. I disabled IPv6, Changed Avahi Settings, Changed Homebridge Versions, Changed Node Versions, Resetted Homebridge Completely ... nothing helped. I realized trough some troubleshooting that in the Bonjour Browser homebridge disappeared after some time ... and came back online ... and dissappeared. I read nearly all issues regarding "No Response" and someone (I don't find you anymore but thank you stranger) said that disabling "IGMP Snooping" in the Netgear Switch Settings helped to fix the problem. 

In the second I turned it off ALL devices came back online immediately. It was just like the TCP-Packets where waiting for that setting to be switched off 😄 

Sorry for creating a "new issue" but even if this Report gets closed instantly I hope I can help someone who is searching in the closed issues for the same problem because I don't want that anybody has to spend hours of debugging to find out that it is an issue with a network switch like me. Maybe someone could add a hint to the Wiki?

Just in the End: Thank you for Homebridge. I used it years ago but switched away very quickly because of reliability issues but 6 months ago I started using it again and if you exclude that little problem (which is not your fault) I had while switching I had not a single problem with homebridge and I enjoy using it everyday!

TL;DR If you have a Netgear switch and not responding devices disable "IGMP Snooping" in the switch settings it worked for me!

**Comment by Supereg** [comment]
You are not the first person telling me that turning off IGMP snooping helped them. I would assume that IGMP is covered by the node network stack or by the kernel itself 🤔
Its probably worth investing why this issue occurred.
Just for ht records, would you be able to supply which Kernel version and nodejs version you were using?

**Comment by christiantschoerner** [comment]
> 
> 
> You are not the first person telling me that turning off IGMP snooping helped them. I would assume that IGMP is covered by the node network stack or by the kernel itself 🤔
> Its probably worth investing why this issue occurred.
> Just for the records, would you be able to supply which Kernel version and nodejs version you were using?
> 
> EDIT: got a netgear switch with IGMP snooping, I might just try to reproduce the issue 🤔

It would be awesome if you could try it! I have a GS116Ev2 (Main Switch attached to router - On this switch the setting was enabled) and a GS105Ev2 (which is plugged into the main switch - it was disabled there)

**Comment by RoganDawes** [comment]
I have been experiencing a similar (perhaps not identical) issue. I am using Home Assistant with the homekit integration. I can add the integration to home assistant, then scan the QR code with my iphone, and am prompted to add all of the lights and switches I chose to export.
However, once that is done, none of the entities are usable - they are all "not responding".
I *do* have a Netgear switch on my network (but not in the path between my HA/Homekit and my various WiFi devices or phone), and while IGMP snooping WAS previously enabled, disabling it had no effect at all.
I did have Homekit integration working at one point, but as I don't actually use it regularly, I have no idea when it stopped working, to try to narrow the cause down.

**Comment by rainman1962** [comment]
I have this same problem.  However, disabling IGMP snooping on my Netgear router did not help (it was already disabled!).  What did work (so far) was to ssh into my Pi and do the following:

1.  Add the line dtoverlay=disable-wifi to /boot/config.txt.  This will turn off WIFI on the pi.
2. Confirm that it is off with ifconfig.
3. Plug in Pi to ethernet
4. reboot pi
5. (may be optional) see what IP eth0 pi is using and Reserve that IP on my Netgear router 
6. reboot router

voila...no more Not Responding!  Seems like for some reason, when pi was connecting to WIFI I would get this problem but when I forced it to a wired connection, it went away...so far (30 minutes and counting).
